src/app/actions/adminAction.ts

```
@@ -108,332 +109,189 @@ export const insertNewSessionFromAdmin = async (
+                          (await prismaTx.gameStat.findFirst({
+                            where: {
+                              statName: playerStat.stat,
+                              gameId: sessionGame.gameId,
+                            },
+                          })) || undefined;
+                        if (gameStat)
+                          gameStatMap.set(playerStat.stat, gameStat);
+                      }
+
+                      // If gameStat is still not found, throw an error
+                      if (true)
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)